### PR TITLE
Point installation links to new installation guide in docs

### PR DESCRIPTION
**Pull request by fonnesbeck**
Change install guide links to https://www.pymc.io/projects/docs/en/latest/installation.html  instead of wiki

### CODE REVIEW

File: docs/source/learn/core_notebooks/pymc_overview.ipynb [hunk 1]
```diff
@@ -43,7 +43,7 @@
     "\n",
     "Running PyMC requires a relatively recent Python interpreter, preferrably version 3.8 or greater. A complete Python installation for Mac OSX, Linux and Windows can most easily be obtained by downloading and installing the free [`Anaconda Python Distribution`](https://store.continuum.io/cshop/anaconda/) by ContinuumIO or the open source [Miniforge](https://github.com/conda-forge/miniforge).\n",
     "\n",
-    "Once Python is installed, follow the  {ref}`platform-specific installation guides <installation>` on the PyMC GitHub repository.\n",
+    "Once Python is installed, follow the [insallation guide](https://www.pymc.io/projects/docs/en/latest/installation.html) on the PyMC documentation site.\n",
```
Comment (OriolAbril) [inline docs/source/learn/core_notebooks/pymc_overview.ipynb:46]: I think that should continue to be a sphinx-cross reference, we should only remove the platform-specific note

Comment (michaelosthege) [reply docs/source/learn/core_notebooks/pymc_overview.ipynb:46]:
```suggestion
    "Once Python is installed, follow the {ref}`installation guide <installation>` on the PyMC documentation site.\n",
````
@OriolAbril like this? Can you commit it it correct? Let's get this merged

Comment (OriolAbril) [reply docs/source/learn/core_notebooks/pymc_overview.ipynb:46]: I edited the comment. I am on my mobile and it won't let me commit though, some refresh issue, probably buggy internetI am on the train. I removed the prural in guides as there are no longer multiple guides

File: README.rst [hunk 2]
```diff
@@ -62,12 +62,7 @@ Audio & Video
 Installation
 ============
 
-To install PyMC on your system, follow the instructions on the appropriate installation guide:
-
--  `Installing PyMC on MacOS <https://github.com/pymc-devs/pymc/wiki/Installation-Guide-(MacOS)>`__
--  `Installing PyMC on Linux <https://github.com/pymc-devs/pymc/wiki/Installation-Guide-(Linux)>`__
--  `Installing PyMC on Windows <https://github.com/pymc-devs/pymc/wiki/Installation-Guide-(Windows)>`__
-
+To install PyMC on your system, follow the instructions on the `installation guide <https://www.pymc.io/projects/docs/en/latest/installation.html>`__.
```
Comment (OriolAbril) [inline README.rst:65]: I would link to stable docs, not to latest. But unlike the comment/change below, this one should not be blocking

Comment (michaelosthege) [reply README.rst:65]: Nah, generally I'd agree, but for installation instructions in the past we often had to respond to changes/breaks in dependencies quickly, so I'd rather have people look at the latest up to date version.

File: docs/source/learn/core_notebooks/pymc_overview.ipynb (hunk 1 above)
Comment (michaelosthege) [reply docs/source/learn/core_notebooks/pymc_overview.ipynb:46]: I had the same problems. Had to do the commit locally..

**Pull request merged**